### ISSUE TITLE
Remove Cobra mousetrap for non-windows

### ISSUE THREAD

**Issue by moorereason** [spf13/hugo#1710]
https://github.com/spf13/cobra/pull/197 breaks Hugo compilation on non-windows.

**Comment by levelonedev** [comment]
I think I am running into the same problem using freebsd

```
O_O[christopher@hugo:~]$ go get -v github.com/spf13/hugo
github.com/spf13/hugo/commands
# github.com/spf13/hugo/commands
go/src/github.com/spf13/hugo/commands/hugo.go:201: undefined: cobra.MousetrapHelpText
```

FreeBSD 9.3-RELEASE-p28
go version go1.5.1 freebsd/amd64

**Comment by anthonyfok** [comment]
Thanks for the report!
Fixed in 9c39593 (master) and 1210519 (v0.15.docs).
